### ISSUE TITLE
create the number of visitors section & also make the copy right year as dynamic with current year 

### ISSUE THREAD

**Issue by Mohit5Upadhyay** [Mohit5Upadhyay/WeatherApplication#48]
make the number of visitors section on the webpage in the footer section it must be dynamic give counting how many users visits the web page and also make the copy right as dynamic , it extract current year from date .

**Comment by ZoyaYusuf** [comment]
kindly assign me. 
Could you please add gssoc-exd and level lables als well

**Comment by Mohit5Upadhyay** [comment]
Yes , @ZoyaYusuf go ahead to implement the functionality as prescribed above in a UI/UX friendly manner

**Comment by ZoyaYusuf** [comment]
Sir, I have made the copyright year dynamic. 
but, for counting the number of visitors, we need to use Backend - technologies like Node.js 
Can i work with that?

**Comment by Mohit5Upadhyay** [comment]
Yes , you can implement it .

**Comment by ZoyaYusuf** [comment]
I just saw that you have deployed this project using github pages. And **GitHub Pages does not support server-side code like Node.js or databases.** 
All client-side solutions are tied to the device and browser. Each time a user accesses the website from a different device or browser, they are considered a new visitor and the count is lost.
Are there any suggestions for what i can do? 
one way is that we can **use third-party services with GitHub Pages** is an easy way to display a visitor count without having to build your own backend. Services like **Hitwebcounter, FreeCounter, and Google Analytics** are reliable options. They handle the data storage and provide you with the necessary code to integrate the counter into your static GitHub Pages site.

**Comment by ZoyaYusuf** [comment]
@Mohit5Upadhyay Kindly tell me what should i do now. 
Should pull request for changing the copyright year only?
